core/scheduler/scheduler.go

```diff
@@ -114,7 +114,7 @@ func (s *Scheduler) Run() error {
 			return nil
 		case slot := <-slotTicker:
 			slotCtx := log.WithCtx(ctx, z.I64("slot", slot.Slot))
-			log.Debug(slotCtx, "Slot ticked")
+			log.Info(slotCtx, "Slot ticked")
```

Comment:
mmmm, not sure about this, lets leave as debug for now.

Reply:
sure